docs/resources/Guild.md

```diff
@@ -695,6 +700,7 @@ Modify a guild's settings. Requires the `MANAGE_GUILD` permission. Returns the u
 | features                      | array of [guild feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features) strings | enabled guild features                                                                                                                                            |
 | description                   | ?string                                                                             | the description for the guild                                                                                                                                     |
 | premium_progress_bar_enabled  | boolean                                                                             | whether the guild's boost progress bar should be enabled                                                                                                          |
+| safety_alerts_channel_id      | ?snowflake                                                                          | the id of the channel where admins and moderators of Community guilds receive safety alerts from Discord                                                          |
```

Comment:
I'm not able to set this to null once set 🤔

Reply:
any update on this?

Reply:
yea this seems to follow existing pattern of at least one other channel field where the way to set it to `null` is by passing in the guild id. I'll look into changing it so that passing `null` is enough.

Reply:
updated so you can pass `null`, will go live either later today or Monday

Reply:
tysm hemu!